activities/community-calls/post-process-community-call.md

```diff
@@ -1,6 +1,8 @@
 ---
 type: Guide
 explains: How to post-process a community call
+redirect_from:
+    - activities/codebase-stewardship/community-calls/post-process-community-call
 ---
 
 # Post-processing a community call
```

Comment:
This section has links that is insufficient now when it is not specifically for the Standard community call anymore.